app/assets/javascripts/staff/program/grid.js

```diff
@@ -190,7 +190,7 @@
     if (url == null || url.length==0) {
       return;
     }
-
+    console.log('url:', url)
```

Comment:
Generally, remove `console.log`'s unless it's a case where something needs debugging on prod.